suite-native/module-home/src/screens/HomeScreen/components/PortfolioContent.tsx

```diff
@@ -42,31 +32,14 @@ export const PortfolioContent = forwardRef<PortfolioGraphRef>((_props, ref) => {
                     <Assets />
                 </Box>
                 {isPortfolioTrackerDevice && (
```

Comment:
The `IsDeviceConnectEnabled` FF is configured in such a way that using `!isUsbDeviceConnectFeatureEnabled` hides the Recieve button in Simulator even though it's visible in the dev app.